src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,31 @@
+import { useContext } from 'react';
+import { PersonItem } from '../PersonItem';
+import { PeopleContex } from '../../peopleContext';
```

Comment:
There is a typo in your context import. It should be `PeopleContext` instead of `PeopleContex`.